p/psad/psad_UBI-8.3.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+# ----------------------------------------------------------------------------
+#
+# Package       : psad
+# Version       : 3.0
+# Source repo   : https://github.com/mrash/psad
+# Tested on     : UBI 8.2
+# Script License: GPL-2.0 License  
+# Maintainer    : Nishikant Thorat <nthorat@us.ibm.com>
+#
+# Disclaimer: This script has been tested in non-root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+yum install -y git wget vim
+yum install -y make
+yum install -y gcc
+yum install -y initscripts chkconfig
+yum install -y iptables iproute
+yum install -y net-tools
+yum install -y  ruby-devel gcc make rpm-build rubygems
+gem install --no-document fpm
+# Install iptable perl package
+wget http://www.cipherdyne.org/modules/IPTables-ChainMgr-1.6.tar.gz
+tar -xzf IPTables-ChainMgr-1.6.tar.gz
+cd IPTables-ChainMgr-1.6
+perl Makefile.PL
+yum install -y  perl-ExtUtils-MakeMaker
+make
+make install
+cd ..
+# Install cpan and Date perl package using cpan
+yum install -y cpan
+export PERL_MM_USE_DEFAULT=1; cpan Date::Calc 
+
+git clone https://github.com/mrash/psad
+cd psad
+make all 
+cd test 
+./test-psad.pl
+rm -rf psad-install/var/log/psad/*
+fpm -s dir -t rpm -C psad-install/ --version  $(cat ../VERSION) -n psad
```

Comment:
Can we put a comment stating that this will generate an RPM, specify its location and / or provide the command to install it?

Reply:
Done(Added location in comment)